scripts/update-abi-registry.js

```diff
@@ -25,7 +31,7 @@ async function fetchNodeVersions () {
     const lts = metadata.hasOwnProperty('lts') ? [metadata.lts, metadata.maintenance] : false
     versions[version] = {
       runtime: 'node',
-      target: version,
+      target: targetPinning[version] ?? version,
```

Comment:
issue: the nullish coalescing operator was added in Node 14, according to https://node.green/ and [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) - if we use this, then **this PR creates a breaking change**, `engines` in `package.json` needs to be updated appropriately, and there needs to be a major version bump.